src/tacacs/nss/patch/0010-Send-remote-address-in-TACACS-authorization-message.patch

```diff
@@ -0,0 +1,114 @@
+From 7228127e9e846b834e5ec81416763e9a607f253e Mon Sep 17 00:00:00 2001
+From: liuh-80 <liuh@microsoft.com>
+Date: Tue, 25 Oct 2022 10:34:08 +0800
+Subject: [PATCH] Send remote address in TACACS+ authorization message.
+
+---
+ nss_tacplus.c | 69 ++++++++++++++++++++++++++++++++++++++++++++++++++-
+ 1 file changed, 68 insertions(+), 1 deletion(-)
+
+diff --git a/nss_tacplus.c b/nss_tacplus.c
+index 2de00a6..d59869b 100644
+--- a/nss_tacplus.c
++++ b/nss_tacplus.c
+@@ -33,6 +33,7 @@
+ #include <ctype.h>
+ #include <netdb.h>
+ #include <nss.h>
++#include <limits.h>
+ 
+ #include <libtac/libtac.h>
+ 
+@@ -717,6 +718,66 @@ connect_tacacs(struct tac_attrib **attr, int srvr)
+     return fd;
+ }
+ 
++/*
++ * Get environment variable first part by name and delimiters
++ */
++int get_environment_variable_first_part(char* dst, socklen_t size, const char* name, const char* delimiters)
```

Comment:
>get_environment_variable_first_part [](http://example.com/codeflow?start=5&length=35)

all the functions return values are not well defined in function level comment.

You have some choices:
1. use well defined name enum/constants
2. define meaning of the number in function level comment.

Reply:
Fixed, add constants for return values.